pandas/util/_tester.py

```diff
@@ -3,11 +3,12 @@
 """
 import os
 import sys
+from typing import NoReturn
 
 PKG = os.path.dirname(os.path.dirname(__file__))
 
 
-def test(extra_args=None):
+def test(extra_args=None) -> NoReturn:
```

Comment:
do we use this pattern elsewhere?

Reply:
I don't think so, but is the correct one because of ``sys.exit`` is used here, if I underdtand it correctly.

Reply:
Hmm, I see now NoReturn was only added in Python 3.6.2. We support 3.6.1 currently.

I'll remove this annotation unless peope think it's good idea to raise the python minimum version (I don't think it's a good idea myself).